packages/crispin/docs/index.mdx

```diff
@@ -0,0 +1,67 @@
+# Crispin
+Crispin is heavily inspired by the fantastic [WinstonJS](https://github.com/winstonjs/winston) project!
+Thus, the name of this project is a nod to the Winston project by way of another reference.
+In the video game Overwatch the character Winston is voiced by Crispin Freeman.
+
+> Don't comment out your logs (ever again). Use tests and configure loggers instead.
+
+## Install
+
+```shell
+flutter pub add crispin
+```
+
+## Basic Usage
+Crispin acts as a "Singleton" so you can configure it once and then use it throughout your application without ever needing comment out lines of code.
```

Comment:
Should we link `Singleton` to [this](https://en.wikipedia.org/wiki/Singleton_pattern) or would that be over kill?

Reply:
@Rohithgilla12 going with https://dart.academy/creational-design-patterns-for-dart-and-flutter-singleton/ for now